gtsfm/scene_optimizer.py

```diff
@@ -366,3 +366,24 @@ def aggregate_frontend_metrics(
         num_valid_entries,
         num_entries,
     )
+
```

Comment:
should we add this as part of this PR?

Reply:
either way, up to you, i can put it in a separate one if you prefer